gap/pcpgrp/fitting.gi

```diff
@@ -56,6 +56,7 @@ end );
 ##
 InstallMethod( FittingSubgroup,
                "for pcp groups", [IsPcpGroup],
+			   777, # Prevent methods using intersection being ranked higher
```

Comment:
This comment does not make it clear that this is about a method from the GAP library (it could also be a second method defined in the polycyclic package; or defined in a third package). And that it uses intersection is secondary, isn't it?
```suggestion
			   SUM_FLAGS, # Prevent generic GAP library method for finite groups being ranked higher
```